src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -160,11 +160,13 @@
         Document coverSheet = pdfManagementService.generateAndUpload(callbackRequest, DocumentType.GRANT_COVER);
         log.info("Generated and Uploaded cover document with template {} for the case id {}",
                 DocumentType.GRANT_COVER.getTemplateName(), callbackRequest.getCaseDetails().getId().toString());
-
+        Document letterOfGrantIssuedState = null;
```

Comment:
why did you call it letterOfGrantIssuedState? isnt it just letterOfGrantIssued? and the other type is letterOfGrantIssuedIntestacy. For Doc Types too